recipe/meta.yaml

```diff
@@ -11,38 +11,36 @@ source:
       - noavx2.patch
 
 build:
-  number: 8
+  number: 9
   entry_points:
     - pt2to3 = tables.scripts.pt2to3:main
     - ptdump = tables.scripts.ptdump:main
     - ptrepack = tables.scripts.ptrepack:main
     - pttree = tables.scripts.pttree:main
 
 requirements:
+  host:
+    - {{ compiler("c") }}
   build:
     - python
     - setuptools
     - cython
-    - numpy 1.8.*  # [not (win and (py35 or py36))]
-    - numpy 1.9.*  # [win and py35]
-    - numpy 1.11.*  # [win and py36]
-    - zlib 1.2.11
-    - lzo 2.*  # [linux]
-    - bzip2 1.0.*
-    - hdf5 1.10.1
+    - numpy
+    - zlib
+    - lzo  # [linux]
```

Comment:
FWIW we have macOS and Windows builds of `lzo` as well. Need not explore them here. Just providing the information.